src/main/java/org/infernalstudios/shieldexp/events/ClientEvents.java

```diff
@@ -30,13 +38,8 @@
 import net.minecraftforge.fml.RegistryObject;
 import net.minecraftforge.fml.common.Mod;
 
-@Mod.EventBusSubscriber(modid = ShieldExpansion.ID, bus = Mod.EventBusSubscriber.Bus.MOD, value = Dist.CLIENT)
 public class ClientEvents {
     public static void setup() {
-        MinecraftForge.EVENT_BUS.register(new ClientEvents());
-        MinecraftForge.EVENT_BUS.register(new FovEvent());
-        MinecraftForge.EVENT_BUS.register(new TooltipEvents());
-
         initShields();
     }
```

Comment:
This can be rewritten as:
```java
@SubscribeEvent
public void clientSetup(FMLClientSetupEvent event) {
  initShields();
}
```
From `ShieldExpansion::clientSetup`. The DistExecutor isn't necessary btw, since the event is only fired on the client anyways.

Reply:
This can also be applied to `ShieldExpansion::commonSetup`, by moving it to `ShieldExpansionEvents.java`

Reply:
> This can also be applied to `ShieldExpansion::commonSetup`, by moving it to `ShieldExpansionEvents.java`

Sorry but failed to guess what you mean "it" pls?

Reply:
Just like you moved `clientSetup` to `ClientEvents`, you can do the same with `commonSetup` and `ShieldExpansionEvents`